### ISSUE TITLE
fastq-dump does not respect caching configuration

### ISSUE THREAD

**Issue by standage** [ncbi/sra-tools#32]
I'm working on a cluster with a very small `$HOME` partition, so I recently learned the hard way about the caching behavior of `fastq-dump`. I'm sure this behavior is very useful in some cases, but I think the vast majority of users would be better served if this were an opt-in feature rather than the default. But I digress...

While I was searching the documentation to figure out how to change the default cache directory, I noticed that caching can be disabled. At least, that's the claim. However, even after disabling caching with `vdb-config`, `fastq-dump` still creates large cache files.

I'm using version 2.6.2. This is the `~/.ncbi/user-settings.mkfg` generated by `vdb-config`.

```
## auto-generated configuration file - DO NOT EDIT ##

/config/default = "false"
/krypto/pwfile = "$(NCBI_HOME)/vdb-passwd"
/repository/user/cache-disabled = "true"
/repository/user/default-path = "/home/standage/ncbi"
/repository/user/main/public/root = "/scratch/standage/sra-cache"
```

**Comment by kwrodarmer** [comment]
This issue began with release 2.6.2, and was still there in 2.6.3. We are about to release 2.7.0 that fixes the problem.

**Comment by kwrodarmer** [comment]
Please confirm that you no longer have this issue, and that I can close.

**Comment by standage** [comment]
I can confirm this has been fixed with release 2.7.0. Thanks!

**Comment by kwrodarmer** [comment]
Just a note that we will be issuing a new release to comply with [OMB memorandum M-15-13](https://www.whitehouse.gov/sites/default/files/omb/memoranda/2015/m-15-13.pdf) that mandates converting all Federal services to use HTTPS. So please stay tuned for a release in early September. Thanks!
